flink-mesos/pom.xml

```diff
@@ -0,0 +1,203 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>flink-parent</artifactId>
+		<groupId>org.apache.flink</groupId>
+		<version>0.10-SNAPSHOT</version>
+		<relativePath>..</relativePath>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>flink-mesos</artifactId>
+	<name>flink-mesos</name>
+	<packaging>jar</packaging>
+
+	<properties>
+		<mesos.version>0.23.0</mesos.version>
+		<unfiltererd.version>0.8.4</unfiltererd.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>com.github.scopt</groupId>
+			<artifactId>scopt_${scala.binary.version}</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>net.databinder</groupId>
+			<artifactId>unfiltered-jetty_${scala.binary.version}</artifactId>
```

Comment:
The dependency is for the simple http server for hosting the configuration for the executors.

The unfiltered library is under mit license. 

Sent from my iPhone
> On Oct 11, 2015, at 20:28, Henry Saputra <notifications@github.com> wrote:
> 
> In flink-mesos/pom.xml:
> 
> > +	<name>flink-mesos</name>
> > +	<packaging>jar</packaging>
> > +
> > +	<properties>
> > +		<mesos.version>0.23.0</mesos.version>
> > +		<unfiltererd.version>0.8.4</unfiltererd.version>
> > +	</properties>
> > +
> > +	<dependencies>
> > +		<dependency>
> > +			<groupId>com.github.scopt</groupId>
> > +			<artifactId>scopt_${scala.binary.version}</artifactId>
> > +		</dependency>
> > +		<dependency>
> > +			<groupId>net.databinder</groupId>
> > +			<artifactId>unfiltered-jetty_${scala.binary.version}</artifactId>
> What is this dependency for? And also what is the license for it?
> 
> —
> Reply to this email directly or view it on GitHub.
>